src/main/java/pl/cinema/server/screening/query/ScreeningRepositoryPort.java

```diff
@@ -0,0 +1,11 @@
+package pl.cinema.server.screening.query;
+
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+
+public interface ScreeningRepositoryPort {
+
+    List<ScreeningQuery> getAvailableScreenings(LocalDateTime start, LocalDateTime end);
+    Optional<ScreeningQuery> getScreeningDetails(Long id);
```

Comment:
zwracanie `Query`? spodziewałby się bardziej, że `Query` to jest coś co jest parametrem metody, nie jej zwrotką